CodeStarterCamp_Week1/PpePpeRo[Step3].swift

```diff
@@ -2,7 +2,49 @@
 //  PpePpeRo[Step3].swift
 //  CodeStarterCamp_Week1
 //
-//  Created by 임경인 on 2022/04/21.
+//  Created by 임경인 on 2022/04/26.
 //
 
 import Foundation
+
+func printPpePpeRoBody(high: Int, body: String, toping: String) {
+    for _ in 1...high/2 {
+        print("\(toping)\(body)\n \(body)\(toping)",
+              separator: ""
+        )
+    }
+}
+
+func printPpePpeRoStick(length: Int, stick: String) {
+    for _ in 1...length {
+        print(" \(stick)")
+    }
+    print("                         ")
```

Comment:
```suggestion
    print()
```

이렇게 해도 덤덤님이 원하는 결과가 나올 것 같아요 😊